dags/pipeline_small_multiprocessing_old.py

```diff
@@ -0,0 +1,322 @@
+if OTAP_ENVIRONMENT.lower().endswith("ont"):
+    NUM_WORKERS = 8
+else:
+    NUM_WORKERS = 16
+
+# Command that you want to run on container start
+DAG_ID: Final = "cvt-pipeline-small_multiprocessing_old"
+DATATEAM_OWNER: Final = "cvision2"
+DAG_LABEL: Final = {"team_name": DATATEAM_OWNER}
+AKS_NAMESPACE: Final = os.getenv("AIRFLOW__KUBERNETES__NAMESPACE")
+AKS_NODE_POOL: Final = "cvision2work"
+
+# List here all environment variables that also needs to be
+# used inside the K8PodOperator pod.
+GENERIC_VARS_NAMES: list = [
+    "USER_ASSIGNED_MANAGED_IDENTITY",
+    "AIRFLOW__SECRETS__BACKEND_KWARGS",
+    "AIRFLOW_CONN_POSTGRES_DEFAULT"
+]
+
+
+client_id = os.getenv("USER_ASSIGNED_MANAGED_IDENTITY")
+credential = ManagedIdentityCredential(client_id=client_id)
+blob_service_client = BlobServiceClient(account_url=BLOB_URL,
+                                       credential=credential)
+
+
+def get_generic_vars() -> dict[str, str]:
+    """Get generic environment variables all containers will need.
+
+    Note: The K8PodOperator spins up a new node. This node needs
+        to be fed with the nessacery env vars. Its not inheriting
+        it from his big brother/sister/neutral the worker pod.
+
+    :returns: All (generic) environment variables that need to be included into the container.
+    """
+    GENERIC_VARS_DICT: dict[str, str] = {
+        variable: os.environ[variable] for variable in GENERIC_VARS_NAMES
+    }
+    return GENERIC_VARS_DICT
+
+
+def remove_unblurred_images(**context):
+    """
+    TODO replace looping through images, find faster implementation
+    Remove unblurred images from storage account after we:
+    - blurred them
+    - stored the metadata in the postgres database
+    """
+    date_ = context["dag_run"].conf["date"]  # retrieve date again since it works differently with python operator
+    start_date = datetime.strptime(date_, "%Y-%m-%d %H:%M:%S.%f")
+    my_format = "%Y-%m-%d_%H-%M-%S"
+    start_date_dag = start_date.strftime(my_format)
+    blob_list = blob_service_client.get_container_client(container="unblurred").list_blobs()
+    counter = 0
+    for blob in blob_list:
+        if blob.name.split("/")[0] == start_date_dag:  # only delete images from one date
+            todelete_blob_client = blob_service_client.get_blob_client(container="unblurred", blob=blob.name)
+            todelete_blob_client.delete_blob()
+            counter = counter + 1
+
+    print(f"Successfully deleted {counter} files!")
+
+
+def count_blobs_in_container(container: str) -> int:
+    # Blob storage API does not support querying blob count
+    blob_list = blob_service_client.get_container_client(container=container).list_blobs()
+    return len([x for x in blob_list])
+
+
+default_args = {
+    'depends_on_past': False,
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 0,
+    'retry_delay': timedelta(minutes=5),
+}
+
+with DAG(
```

Comment:
you can remove this trigger dag. we won't need it to generate batch jobs anymore